src/include/distributed/multi_executor.h

```diff
@@ -74,13 +74,26 @@ extern void CitusExecutorRun(QueryDesc *queryDesc, ScanDirection direction, uint
 							 bool execute_once);
 extern void AdaptiveExecutorPreExecutorRun(CitusScanState *scanState);
 extern TupleTableSlot * AdaptiveExecutor(CitusScanState *scanState);
-extern uint64 ExecuteTaskListExtended(RowModifyLevel modLevel, List *taskList,
-									  TupleDesc tupleDescriptor,
-									  Tuplestorestate *tupleStore,
-									  bool hasReturning, int targetPoolSize,
-									  TransactionProperties *xactProperties,
-									  List *jobIdList,
-									  bool localExecutionSupported);
+
+typedef struct ExecutionParams
```

Comment:
Needs a comment describing what this is for. Ideally, we'd also document the fields.

Reply:
@marcocitus can you check the documentation for the fields